### PR TITLE
Completed python functions. - Kevin Beck

### PR DESCRIPTION
**Pull request by kevinmbeck**
Screen Shot 2022-04-20 at 8 35 29 PM
Not sure why there are none after some of them.

### CODE REVIEW

File: ds-prep-python-functions/main.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    print((a-b)**2)
+print(squared_value(12,9))
+def hours_to_minutes(hours):
+  print(hours*60)
+hours_to_minutes(hours = 2)
+def get_circumference(radius):
+  print(radius*2*3.14)
+print(get_circumference(radius=1))
+def linear_transform(x, slope, intercept):
+  print((slope*x)+intercept)
```
Comment (TimelikeClosure) [inline ds-prep-python-functions/main.py]: Please update your named functions so they `return` their values, not `print` them. `print` should not appear anywhere inside the function definitions:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: ds-prep-python-functions/main.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+bool_to_int = lambda value: int(value)
+print(bool_to_int(True))
+get_smaller = lambda a, b: a if (a<b) else b
+print(get_smaller(32,45))
+def cube(base):
+  print(base**3)
+cube(9)
```
Comment (TimelikeClosure) [inline ds-prep-python-functions/main.py]: The `cube` function definition should return its value instead of printing it

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by kevinmbeck** [comment]
Screen Shot 2022-04-20 at 8 49 04 PM

**Comment by kevinmbeck** [comment]
Fixed the return in cube function

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**